l10n_it_fatturapa_out/wizard/efattura.py

```diff
@@ -254,10 +254,30 @@ def to_xml(self, env):
         content = env.ref(
             "l10n_it_fatturapa_out.account_invoice_it_FatturaPA_export"
         )._render(template_values)
-        # 14.0 - occorre rimuovere gli spazi tra i tag
-        root = etree.fromstring(content, parser=etree.XMLParser(remove_blank_text=True))
-        # già che ci siamo, validiamo con l'XMLSchema dello SdI
-        errors = list(fpa_schema.iter_errors(root))
+        try:
+            # 14.0 - occorre rimuovere gli spazi tra i tag
+            root = etree.fromstring(
+                content, parser=etree.XMLParser(remove_blank_text=True)
+            )
+            # già che ci siamo, validiamo con l'XMLSchema dello SdI
+            errors = list(fpa_schema.iter_errors(root))
+        except etree.XMLSyntaxError as e:
+            entry = e.error_log.last_error
+            error_msg = _(
+                "Error processing invoice(s) %(invoices)s.\n\n"
+                "Error message: %(error)s\n\n",
+                invoices=", ".join(
+                    inv.display_name for inv in template_values["invoices"]
+                ),
+                error=entry.message,
+            )
+            # when there is a char belong to
```

Comment:
I commenti sono in italiano nel resto del file. Magari facciamo una passata, e li traduciamo tutti.